File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Declaring Variables
    int height, space , hash;
    
    // Condition for Accepting of Value
    do
    {
       printf("Height:");
       height = get_int(0); 
    }
     while (height < 1 || height > 8);
    
    // loop for row
    for (int i = 0; i < height; i++)
    {
       // loop for space
        for (space = (height - i); space >0; space--)
        {
            printf(" ");
        }
         // loop for hash
        for (hash = 0; hash < (i + 1); hash++)
        {
            printf("#");
        }
        
        printf("\n");
    }
    
}

File: mario.py
from cs50 import get_int

def main():
    while True:
        height = get_int("Height: ")
        n = height
        if n > 0 and n <= 8:
            break

    for i in range(n):
        for j in range(n - i - 1):
            print(" ", end = "")
        for k in range(i+1):
            print("#", end = "")
        print("")

if __name__ == "__main__":
    main()
